lib/please_cli/please_cli/decision_task.py

```diff
@@ -393,6 +393,11 @@ def cmd(ctx,
 
     taskcluster_queue = cli_common.taskcluster.get_service('queue')
 
+    taskcluster_notify = cli_common.taskcluster.get_service('notify')
+    if channel in please_cli.config.DEPLOY_CHANNELS:
+        taskcluster_notify.irc(dict(channel='#release-services',
+                                    message=f'New deployment on {channel} just started'))
```

Comment:
It would be also useful to add a link to the taskcluster group we are creating. But for this we need to move notification after line 408

```suggestion
                                    message=f'New deployment on {channel} is about to start: https://tools.taskcluster.net/groups/{task_group_id}'))
```